Sources/Rendering/OpenGL/PolyDataMapper/index.js

```diff
@@ -1747,7 +1747,11 @@ function vtkOpenGLPolyDataMapper(publicAPI, model) {
         model.drawingEdges =
           drawSurfaceWithEdges &&
           (i === primTypes.TrisEdges || i === primTypes.TriStripsEdges);
-        if (!model.drawingEdges || !model.renderDepth) {
+        // don't draw edges when rendering depth or rendering for picking
```

Comment:
if a polydata has only polylines, shouldn't they be rendered for picking ?

Reply:
Polylines don't share the same buffer as edges:
```
export const primTypes = {
  Start: 0,
  Points: 0,
  Lines: 1,
  Tris: 2,
  TriStrips: 3,
  TrisEdges: 4,
  TriStripsEdges: 5,
  End: 6,
};
```
Polylines are `Lines` and edges are `TrisEdges` or `TriStripsEdges`, hence the definition of `model.drawingEdges` (l. 1747-1749)